### DISCUSSION TITLE
Load More button on galleries

### DISCUSSION THREAD

**Discussion by viktorturnsek** [elementor/elementor#28394] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When having many pictures in a gallery it can take up a lot of space relative to other elements on a page, especially on a mobile device where you have to scroll for a long time through a large gallery. Having to scroll through a large gallery could make the user uninterested in what content might be after the gallery.

### Describe the solution you'd like.

Adding a load more button and only showing the first couple of images in a gallery would solve this issue by making the gallery compact with the possibility to see all images if the user is interested. When the "Load more" button is pressed the gallery expands to a certain size/number of images with a new "Load more" button on the bottom, or the gallery expands to show all the images depending on preference.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by domainservicesk** [comment]
I complete agree with this feature request, especially for the Elementor Pro Gallery, which should be premium, this feature is pretty basic and thus should be added so we do not have to bloat websites with another 3rd party addons. 

Please, if you decide to add this feature, which i strongly advocate for, add it for **both Gallery types - Single and also for Multiple.**
